### ISSUE TITLE
How to retrain on my own 16velodyne data

### ISSUE THREAD

**Issue by khanln** [traveller59/second.pytorch#258]
Hi, you did a very great job, but when I use your pre-trained model to predict my own 16-vlp data, including retrained kitti data, the result is not good. My velodyne installation height is 1.6m, and did not modify the parameters in the second configuration file. So I hope you can give me some suggestions:

1. Do I need to modify any configuration to apply to 16-vlp data?

2. If I want to train my 16-vlp data, how to rewrite a class to modify the data loader, there is no general idea and method(I only have velodyne annotation data).

very appreciate：）

**Comment by MaAmini** [comment]
@khanln Do you find answer for your questions?

**Comment by wanghengmm** [comment]
i want to train mu 16-vlp data too   ，can you tell me you email

**Comment by chasingw** [comment]
@wanghengmm I'm working for train on my own lidar data and labels with lidar coordinates，My email is cchasingw@gmail.com

**Comment by ryontang** [comment]
@MaAmini @khanln Hi, do you guys find some way to solve this problem?
